python/chronos/src/bigdl/chronos/autots/tspipeline.py

```diff
@@ -338,6 +353,107 @@ def load(file_path):
                           best_config=best_config,
                           **data_process)
 
+    def quantize(self,
+                 calib_data,
+                 val_data=None,
+                 metric=None,
+                 conf=None,
+                 framework='pytorch_fx',
+                 approach='static',
+                 tuning_strategy='bayesian',
+                 relative_drop=None,
+                 absolute_drop=None,
+                 timeout=0,
+                 max_trials=1):
+        """
+        Quantization TSPipeline.
+        :param calib_data:
+
+               | 1. data creator:
+               | a function that takes a config dictionary as parameter and
+               | returns a PyTorch DataLoader.
+               |
+               | 2. a bigdl.chronos.data.TSDataset:
+               | the TSDataset should follow the same operations as the training
+               | TSDataset used in `AutoTSEstimator.fit`.
+               |
+               | 3. A torch.utils.data.dataloader.DataLoader object for calibration.
+               | Required for static quantization.
```

Comment:
Users should set the configs correctly (e.g. past_seq_len, ...). They can be found in `TSPipeline._best_config`

Reply:
Required for static quantization. can be moved ahead.

Reply:
4. numpy ndarray, ...